scrapers/TakeVan.yml

```
@@ -0,0 +1,28 @@
+name: "TakeVan"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - takevan.com/video/
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class="videoTitle"]
+      Date: &dateAttr
+        selector: //div[@id="videoDate"]
+        postProcess:
+          - replace:
+              - regex:  "^Added "
+                with: ""
+          - parseDate: January 2, 2006
+      Details: &detailsSel //div[@class="videoDescription"]/text()
```

This gets only a partial description (the first text only)
switch to 
```suggestion
      Details: //div[@class="videoDescription"]
```
to get all the description